firebase_admin/mlkit.py

```diff
@@ -18,8 +18,77 @@
 deleting, publishing and unpublishing Firebase ML Kit models.
 """
 
+import re
+import requests
+import six
+
+from firebase_admin import _http_client
+from firebase_admin import _utils
+
+
+_MLKIT_ATTRIBUTE = '_mlkit'
+
+
+def _get_mlkit_service(app):
+    """ Returns an _MLKitService instance for an App.
+
+    Args:
+      app: A Firebase App instance (or None to use the default App).
+
+    Returns:
+      _MLKitService: An _MLKitService for the specified App instance.
+
+    Raises:
+      ValueError: If the app argument is invalid.
+    """
+    return _utils.get_app_service(app, _MLKIT_ATTRIBUTE, _MLKitService)
+
+
+def get_model(model_id, app=None):
+    mlkit_service = _get_mlkit_service(app)
+    return Model(mlkit_service.get_model(model_id))
+
+
+class Model(object):
+    """A Firebase ML Kit Model object."""
+    def __init__(self, data):
+        """Created from a data dictionary."""
+        self._data = data
+
+    def __eq__(self, other):
+        if isinstance(other, self.__class__):
+            return self._data == other._data # pylint: disable=protected-access
+        else:
+            return False
+
+    def __ne__(self, other):
+        return not self.__eq__(other)
+
+    #TODO(ifielker): define the Model properties etc
+
+
 class _MLKitService(object):
     """Firebase MLKit service."""
 
-    BASE_URL = 'https://mlkit.googleapis.com'
-    PROJECT_URL = 'https://mlkit.googleapis.com/projects/{0}/'
+    PROJECT_URL = 'https://mlkit.googleapis.com/v1beta1/projects/{0}/'
+
+    def __init__(self, app):
+        project_id = app.project_id
+        if not project_id:
+            raise ValueError(
+                'Project ID is required to access MLKit service. Either set the '
+                'projectId option, or use service account credentials.')
+        self._project_url = _MLKitService.PROJECT_URL.format(project_id)
+        self._client = _http_client.JsonHttpClient(credential=app.credential.get_credential())
```

Comment:
Also pass `base_url=self._project_url` to the constructor

Reply:
done